Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -209,4 +209,86 @@ The following example displays the property with the alias `headline` together w
 
 If you'd like to display properties of `settings`, you can access these by `block.settingsData.myPropertyAlias`.
 
-See this forum post to [display an image or add other custom behaviour to your custom backoffice view](https://our.umbraco.com/forum/using-umbraco-and-getting-started/103560-image-in-custom-block-editor-view#comment-323696).
+### Adding custom implementation to your View
+
+To achieve this you need to add a custom angularJS controller to your custom view, using the ng-controller attribute:
+```
+<div ng-controller="customBlockController" ng-click="block.edit()"> 
+    <h2 ng-bind="block.data.headline"></h2>
+    <p ng-bind="block.data.description"></p>
+</div>
+```
+
+Create a folder inside the App_Plugins folder called 'CustomBlockView' or something more meaningful for your implementation.
+
+Inside this folder create two files, one called package.manifest that contains the following:
+```
+{
+  "javascript": [
+    "~/App_Plugins/CustomBlockView/customBlock.controller.js"    
+  ]  
+}
+```
+Umbraco will parse all package.manifest files and load any resources they reference into the backoffice during startup.
+
+and secondly add a javascript file called
+
+**customBlock.controller.js**
+
+(to match the file referenced in your package.manifest file)
+
+this file then should register your 'customBlockController' mentioned in your view ng-controller attribute with Umbraco's angular module's controllers
+```
+angular.module("umbraco").controller("customBlockController", function ($scope) {
+// you can do your custom functionality here!
+}
+```
+#### Example: Displaying an image from a Media Picker
+Your block may enable you to 'pick' an image for use as the background for a particular block or to display as part of the block layout, but if you try to display this image directly in the view from the property `block.data.image` you'll see the unique id and not the image.
```

Comment:
Sentence was a tat too long - tried splitting it up. Hope it still makes sense 🤞

```suggestion
Your block may enable you to 'pick' an image for use as the background for a particular block or to display as part of the block layout. If you try to display this image directly in the view from the property `block.data.image` you'll see the unique id and not the image.
```